portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,116 @@
-#content {
+/* General styling */
+
+html {
+  scroll-behavior: smooth;
+}
+
+h1 {
+  text-align: center;
+}
+
+h2 {
+  text-align: center;
+}
+
+/* Website welcome styling */
+
+#welcome-content {
   margin-left: auto;
   margin-right: auto;
+  margin-bottom: 40px;
   width: 650px;
+  font-family: 'Questrial', sans-serif;
 }
 
-#greeting-container {
+#fact-container {
   margin-top: 20px;
 }
+
+.button {
+  background-color: #4d0066;
```

Comment:
If possible use the 3 character hexadecimal notation.

https://google.github.io/styleguide/htmlcssguide.html#Hexadecimal_Notation